lib/accessly/policy/base_option_1.rb

```diff
@@ -0,0 +1,86 @@
+module Accessly
+  module Policy
+    class BaseOption1
```

Comment:
@armilam So in the first case, you would interact with the policy like `BaseOptions1.new(actors).edit?` or `BaseOptions1.new(actors).edit?(object)` and in the second case you would use it like `BaseOptions2.new(actors).can?(:edit)` or `BaseOptions2.new(actors).can?(:edit, object)`? Would that be correct?

Reply:
oh nevermind... the tests spell it out 😆